src/promptflow/promptflow/_sdk/entities/_chat_group/__init__.py

```diff
@@ -0,0 +1,6 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+# isort: skip_file
```

Comment:
why need these skip line?

Reply:
copied from an existing init, let me remove it.